src/Routes/Routes.tsx

```diff
@@ -48,6 +48,8 @@ const InnerRoutes = ({ appRoutesList }: { appRoutesList: React.ReactNode[] }) =>
     <Routes>
       {appRoutesList}
       <Route path='/' element={tradeRedirect} />
+      {/* Handle legacy /connect-wallet/* routes by redirecting to main app route. We don't expose these anymore, but users may have old bookmarks */}
+      <Route path='/connect-wallet/*' element={tradeRedirect} />
```

Comment:
suggestion: while at it, we could also add the `earn` route which I supposed can be bookmarked too if you want to look at your opportunities (or even redirect it to `/wallet/earn`)

Reply:
🧠 that's actually pretty smort!

f1987d3888

https://jam.dev/c/2fdadeb3-858b-48a3-9e72-c0fc70cf7948